libcontainer/userns/userns_deprecated.go

```diff
@@ -1,8 +1,13 @@
+// Deprecated: use github.com/moby/sys/userns
```

Comment:
Can we just remove libct/userns ?

Reply:
I think the concern is downstream users that import `github.com/opencontainers/runc/libcontainer/userns`. It seems [there are a fair number of places that import this package](https://github.com/search?q=import+%22github.com%2Fopencontainers%2Frunc%2Flibcontainer%2Fuserns%22&type=code) unfortunately.

Reply:
Yes, correct; mostly to give potential users of this a warning that they should use the separate module instead. We can decide how long to keep the alias in place (although it doesn't give a significant amount of maintenance)